common.mk

```diff
@@ -349,6 +349,7 @@ buildroot: optee-os optee-rust
 		$(DEFCONFIG_FTPM) \
 		--br-defconfig out-br/extra.conf \
 		--make-cmd $(MAKE))
+	@source $(RUST_TOOLCHAIN_PATH)/.cargo/env
```

Comment:
Wouldn't this be lost when this line is done?

Reply:
Good catch :) this should be on the same line as `$(MAKE) $(br-make-flags) ...`

Reply:
That's true, I will fix it.